netlify/netlify-plugin-nextjs
# fix: use forced catchall for preview mode

### Summary
For preview mode, instead of generating a forced redirect for every pre-rendered page, instead make the catch-all a forced redirect. This prevents the need to generate very large numbers of redirects.

### Test plan

1. In the demo site, load `/previewTest`. Check the message to see preview mode is disabled
2. Go to `/api/enterPreview`
3. Go back to `/previewTest`, and check that preview mode is enabled.
4. Go to `/api/exitPreview`

Fixes #791
